tests/runtime/scripts/uprobe_pid_check.bt

```diff
@@ -0,0 +1,18 @@
+BEGIN {
+    @count = 0;
+}
+
+uprobe:./testprogs/uprobe_fork_loop:uprobeFunction1 {
+    if (pid == $1) {
+        // make sure the pid is always what we expect
+        if (@count == 10) {
```

Comment:
Just checking that I understand this correctly: `uprobe_fork_loop` creates 2 processes with different PIDs running `uprobeFunction1` and we check here that we hit the function from wanted PID 10 times before hitting the function from the other PID, which effectively means that we're not attached to the wrong PID.

Sounds a reasonable approach to me, maybe we could just add a comment documenting it somewhere? I can totally see myself staring into it in a couple of months and wondering what the hell is going on :-)

Reply:
> we check here that we hit the function from wanted PID 10 times before hitting the function from the other PID

In this test we should never hit the branch for the other PID. The counting till 10 is just kind of a delay to ensure we never hit the other pid during that time. But I'll add a comment to make that more clear.